src/components/footer/getLocationCookie.ts

```diff
@@ -1,10 +1,13 @@
+import type { CountryCode } from '@guardian/libs';
+
 let countryCode: string | null = null;
 
-export const getGeoLocation = (): string | null => {
+export const getGeoLocation = (): CountryCode | null => {
   if (countryCode === null) {
-    countryCode = getCookie("GU_geo_country");
+    countryCode = getCookie("GU_geo_country") as CountryCode;
```

Comment:
should we be casting this as `CountryCode`? I'm not sure we can guarantee that the string will actually be a CountryCode, and the `getCountryCode` function which consumes the return value of `getGeoLocation` doesn't actually assume that it'll be passed a CountryCode (i.e. it looks like it will return `null` if the `countryCode` param is a string that's not a valid country code). So it feels like casting here might give a false sense of security?